CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+# Release 0.18.3
+- [GH-1658] Shouldn't wfl.service.rawls/batch-upsert handle Boolean attributes? ([#607](https://github.com/broadinstitute/wfl/pull/607))
+
```

Comment:
I believe this release also includes this PR: https://github.com/broadinstitute/wfl/pull/606

```
- [GH-1653] TDR snapshot direct links have changed ([#606](https://github.com/broadinstitute/wfl/pull/606))
```

Reply:
It does!